Cakebrew/BPHomebrewInterface.m

```diff
@@ -92,30 +89,101 @@ - (void)dealloc
 	[[NSNotificationCenter defaultCenter] removeObserver:self];
 }
 
+- (void)cleanup
+{
+  [self.task terminate];
+}
+
+- (BOOL)systemHasAppNap
+{
+  static BOOL systemHasAppNap;
+  if (!systemHasAppNap) {
```

Comment:
`systemHasAppNap` is not a pointer. If the system does not have AppNap, this if will be executed every time. You could replace it for an NSNumber though.